libs/net/dal/Services/ContentService.cs

```diff
@@ -26,8 +32,12 @@ public class ContentService : BaseService<Content, long>, IContentService
     public ContentService(TNOContext dbContext,
         ClaimsPrincipal principal,
         IServiceProvider serviceProvider,
+        IConfiguration config,
+        IElasticClient client,
         ILogger<ContentService> logger) : base(dbContext, principal, serviceProvider, logger)
     {
+        _client = client;
+        _unpublishedIndex = config["ElasticSearch:UnpublishedIndex"] ?? "unpublished_content";
```

Comment:
Do not user magic strings.  Use compile time safe `IOptions` like we use everywhere else.
Also throw an example if there is no property configured.

Reply:
Updated, thanks